charts/camunda-platform-alpha/openshift/values.yaml

```diff
@@ -1,247 +1,19 @@
-      fsGroup: null
-    containerSecurityContext:
-      runAsUser: null
-
-postgresql:
-  global:
-    compatibility:
-      ## Compatibility adaptations for Openshift
-      openshift:
-        ## Adapt the securityContext sections of the deployment to make them compatible with Openshift restricted-v2 SCC: remove runAsUser, runAsGroup and fsGroup and let the platform use their allowed default IDs. Possible values: auto (apply if the detected running cluster is Openshift), force (perform the adaptation always), disabled (do not perform adaptation)
-        adaptSecurityContext: force
-  primary:
-    podSecurityContext:
-      fsGroup: null
-    containerSecurityContext:
-      runAsUser: null
-  readReplicas:
-    podSecurityContext:
-      fsGroup: null
-    containerSecurityContext:
-      runAsUser: null
-  metrics:
-    podSecurityContext:
-      fsGroup: null
-    containerSecurityContext:
-      runAsUser: null
-
-#
-# Zeebe.
-#
-
-# omit the values below if zeebe.enabled is false
-zeebe:
-  configMap:
-    defaultMode: 0555
-  podSecurityContext:
-    fsGroup: null
-  containerSecurityContext:
-    runAsUser: null
-
-# omit the values below if zeebe-gateway.enabled is false
-zeebeGateway:
-  configMap:
-    defaultMode: 0444
-  podSecurityContext:
-    fsGroup: null
-  containerSecurityContext:
-    runAsUser: null
-
-#
-# Console
-#
-# omit the values below if console.enabled is false
-console:
-  podSecurityContext:
-    fsGroup: null
-  containerSecurityContext:
-    runAsUser: null
+global:
+  compatibility:
+    ## Compatibility adaptations for Openshift
+    openshift:
+      ## Adapt the securityContext sections of the deployment to make them compatible with Openshift restricted-v2 SCC: remove runAsUser, runAsGroup and fsGroup and let the platform use their allowed default IDs. Possible values: auto (apply if the detected running cluster is Openshift), force (perform the adaptation always), disabled (do not perform adaptation)
+      adaptSecurityContext: force
 
 #
 # Elasticsearch.
 #
-
-# omit the values below if elasticsearch.enabled is false
 elasticsearch:
-  global:
-    compatibility:
-      ## Compatibility adaptations for Openshift
-      openshift:
-        ## Adapt the securityContext sections of the deployment to make them compatible with Openshift restricted-v2 SCC: remove runAsUser, runAsGroup and fsGroup and let the platform use their allowed default IDs. Possible values: auto (apply if the detected running cluster is Openshift), force (perform the adaptation always), disabled (do not perform adaptation)
-        adaptSecurityContext: force
-  master:
-    podSecurityContext:
-      fsGroup: null
-    containerSecurityContext:
-      runAsUser: null
-  # If needed, OpenShift TuneD operator will be used to set sysctl vm.max_map_count.
-  # Apply the manifest "openshift-tuned.yaml" before install Camunda 8 chart.
-  sysctlImage:
-    enabled: false
   commonLabels:
     tuned.openshift.io/elasticsearch: ""
```

Comment:
The previous issue are fixed.
The current issue I am facing now is this tuned label is needed for the helm upgrade to pass. This label is also set inside the `volumeClaimTemplate` inside elasticsearch. Therefore you are not able to remove it through a helm upgrade. 
I also tried a strategic patch using this command after the helm upgrad:
```bash
kubectl patch statefulset camunda-elasticsearch-master --type='json' -p='[{"op": "remove", "path": "/spec/volumeClaimTemplates/0/metadata/labels/tuned.openshift.io~1elasticsearch"}]'
```
unfortunately, it also did not work. I get the same error:
```
Upgrade "integration" failed: cannot patch "integration-elasticsearch-master" with kind StatefulSet: StatefulSet.apps "integration-elasticsearch-master" is invalid: spec: Forbidden: updates to statefulset spec for fields other than 'replicas', 'ordinals', 'template', 'updateStrategy', 'persistentVolumeClaimRetentionPolicy' and 'minReadySeconds' are forbidden
```

Another way would be to scale down elasticsearch completely, and then upgrade. I do not know if this is an appropriate solution. I have to check with my team.

Reply:
I created a helper function to check if the force keyword is being used on `adapSecurityContext` if the user has set the label then the function doesn't do anything. If the customer has not set this label then the label is set to empty string.